src/desktop/apps/home/components/artists_to_follow/artist_suggestion_query.coffee

```diff
@@ -1,10 +1,16 @@
 module.exports = """
   query($artist_id: String!) {
-    me{
-      suggested_artists(artist_id: $artist_id, exclude_followed_artists: true, exclude_artists_without_artworks: true, size: 5){
-        ... artistCell
+    artist(id: $artist_id) {
+      related {
+        suggestedConnection(excludeFollowedArtists: true, excludeArtistsWithoutArtworks: true, first: 5) {
```

Comment:
This was the place where the schema changed. Instead of this living under `me`, it moved to be part of the `Related` metadata under `Artist`. There's a corresponding update in the client code in https://github.com/artsy/force/pull/5841/files#diff-68dec1726ea72e0d0d4c928106bdb33aR76